docs/core/integration_system_health.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Integration system health
+sidebar_label: "System health"
+---
+
+The system health platform allows integrations to provide information that helps users understand the state of the integration. This can include details such as the availability of an endpoint, the current server that the integration is connected to, how much of a request quota is still available, etc.
```

Comment:
I think we somewhere, either here in the introduction or in the examples below, should clarify the name of the platform module `system_health.py`.